internal/backends/winit/Cargo.toml

```diff
@@ -85,11 +85,11 @@ cocoa = { version = "0.24.0" }
 core-foundation = { version = "0.9.1" }
 core-text = { version = "19.1.0" }
 # For Metal rendering with Skia
-metal = { version = "0.24.0", optional = true }
+metal = { version = "0.24.0" }
```

Comment:
i don't get why all these dependencies are enabled when not building the skia backend.

Reply:
If you prefer I can solve this differently by adding them to the Skia OpenGL build instead. That narrows it down a little bit more.